### DISCUSSION TITLE
VirtualBoxを使ったときのinit.elの設定について

### DISCUSSION THREAD

**Discussion by nakataniShoma** [jsk-lecture/software2#1] [Q&A]
機械工学科4年で学科PCがないため、VirtualBoxでUbuntuを使っています。
emacs用のタイピングソフトの導入をしたいのですが、
init.elの設定が分からないので、教えていただきたいです。

**Comment by k-okada** [comment]
遅くなりました．
init.elは資料の18ページに書いてあるように`~/.emacs.d/` というディレクトリに`init.el`というファイルを作成してください．
```
cd ~/  # ホームディレクトリへ移動
ls -al # ホームディレクトリ以下のファイルを確認，もし ~/.emacs.d というファイルがなければ，，
mkdir ~/.emacs.d # ディレクトリを作成
cd ~/.emacs.d # .emacs.d というデレクトリに移動
emacs -nw init.el または vi init.el # init.el というファイルを作成し編集
```
です．もし，emacs や　vi がインストールされていなければ，16ページの6.3章から読んでみてください．

**Reply by nakataniShoma** [reply]
解決しました。ありがとうございます。
